platform/linuxbsd/key_mapping_x11.cpp

```diff
@@ -294,6 +313,28 @@ static _TranslatePair _scancode_to_keycode[] = {
 	{ Key::SUPER_L, 0x85 },
 	{ Key::SUPER_R, 0x86 },
 	{ Key::MENU, 0x87 },
+	{ Key::F14, 0xC0 },
```

Comment:
It seems to lack F13.

`_WinTranslatePai::_scancode_to_keycode` in `key_mapping_windows.cpp` might be needed to changed. I expect `_scancode_to_keycode()` is reference with Set1 in https://download.microsoft.com/download/1/6/1/161ba512-40e2-4cc9-843a-923143f3456c/translate.pdf. But I don't know about what linux referenced.

Presumably I expect those are mappings for non-USB HID inputs, but I'm not familiar with that.

Reply:
I pushed these changes